external-builds/pytorch/build_prod_wheels.py

```diff
@@ -360,15 +360,28 @@ def do_build(args: argparse.Namespace):
             "Please specify --pytorch-rocm-arch (e.g., gfx942)."
         )
 
+    rocprofiler_path = root_dir / "lib" / "librocprofiler64.so"
+    roctracer_path = root_dir / "lib" / "libroctracer64.so"
```

Comment:
These are only used from the `env` code that I'm not sure about... but if that code is functional, this should be gated on `USE_KINETO` and/or `not is_windows`.